recipe/build.sh

```diff
@@ -0,0 +1,11 @@
+# Copy over the files in include and lib dirs
+for dir in "include" "include/cpp" "lib"
+do
+    for filename in $(basename $(find ${SRC_DIR}/${dir} -type f -maxdepth 1) || exit 1)
+        do mkdir -p ${PREFIX}/${dir} && \

```

Comment:
just as information, it is also possible to create the dir with install (`-D` flag).

Reply:
unfortunately it doesn't seem to be the case on macOS where the default install distribution supports `-d` and not `-D`.

Reply:
hmm, yeah, macOS uses the BSD implementation. would need to add coreutils as a dependency, to have the GNU implementation of install available.

Reply:
yeah, `-d` is good as a direct replacement for `mkdir`. the advantage of -D is that you can just add this to the existing `install` calls and it would already create the missing directories for you. merged code looks very good for me. thanks!